src/service/performance-impl.js

```diff
@@ -121,6 +110,13 @@ export class Performance {
       (this.win.PerformanceObserver &&
         this.win.PerformanceObserver.supportedEntryTypes) ||
       [];
+
+    // If Paint Timing API is not supported, cannot determine first contentful paint
+    if (!supportedEntryTypes.includes('paint')) {
+      this.metrics_['fcp-v'] = new Deferred();
+      this.metrics_['fcp-v'].resolve(null);
```

Comment:
What about `fcp`?

Reply:
There's an existing `fcp` label reported to the CSI that is not offset by the first visible time. Because of this it is probably not the value the analytics user would expect. Therefore we need to add a new value for the analytics user, offset by visible time, which I notated as `fcp-v`. We can't simply override `fcp` as want that to be consistent for any viewers relying on that data

Reply:
Is the `fcpDeferred_` we're removing a `fcp-v`? Or was it `fcp`?

Maybe add a comment on why we're able to resolve `fcp` but not `fcp-v` when `paint` isn't available.

Reply:
Before we were reporting the value of FCP without the visible offset to amp-analytics, so this will change the behavior for amp-analytics but not for

Reply:
The `fcpDeferred_` resolved with `fcp`, so that is a change for amp-analytics users. It is not possible to report `fcp` or `fcp-v` when the paint timing API is not available.